payjoin-cli/src/main.rs

```diff
@@ -14,7 +14,14 @@ fn main() -> Result<()> {
     match matches.subcommand() {
         Some(("send", sub_matches)) => {
             let bip21 = sub_matches.get_one::<String>("BIP21").context("Missing BIP21 argument")?;
-            app.send_payjoin(bip21)?;
+            let fee_rate_sat_per_vb = match sub_matches.get_one::<f32>("fee_rate") {
+                Some(fee_rate) => fee_rate,
+                None => {
+                    log::info!("No fee rate specified, using default of 2.1 sat/vB");
+                    &2.1
+                }
```

Comment:
2.1 sats/vB is a pretty weird default. It begets a setting for either core fallback fee config, non-optional fee_rate param, bitcoin core estimatesmartfee, etc.

I think this should turn into an issue to solve for before we publish a binary of this tool.

Reply:
I changed this to required for now - should I open an issue to get minimum fee rate accepted in mempool?

Reply:
good. yes.